eclipse-scout-core/src/keystroke/KeyStroke.js

```diff
@@ -138,9 +139,19 @@ export default class KeyStroke {
       if (this.field.visible !== undefined && !this.field.visible) {
         return false;
       }
-      // Check enabled state
-      if (this.field.enabled !== undefined && !this.field.enabled) {
-        return false;
+      // Check enabled state (if inheritAccessibility is true)
+      if (!this.inheritAccessibility) {
+        return true;
+      }
+      if (this.field.enabledComputed !== undefined) {
+        if (!this.field.enabledComputed) {
+          return false;
+        }
+      } else if (this.field.enabled !== undefined) {
+        // This should actually not happen because this.field should always be a hypothetical case if this.field is not a widget
+        if (!this.field.enabled) {
+          return false;
+        }
```

Comment:
and here the same: just return this.field.enabled?

is more readable in my opinion and should have the same logic?

Reply:
yep